test/functional/sync_plan/create_test.rb

```diff
@@ -0,0 +1,61 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/sync_plan'
+require_relative 'sync_plan_helpers'
+require_relative '../organization/organization_helpers'
+
+module HammerCLIKatello
+  describe SyncPlan::CreateCommand do
+    include OrganizationHelpers
+    include SyncPlanHelpers
+
+    it 'create with organization ID,name,interval,date and enabled' do
+      api_expects(:sync_plans, :create, 'create a sync plan') do |par|
```

Comment:
@sjha4 sorry, should have mentioned, there's a new syntax (`.with_params`) which makes the test failures much easier to understand. i.e. https://github.com/Katello/hammer-cli-katello/blob/49b48b3c1d431da10c254e3c59e62552cb909122/test/functional/host/extensions/create_test.rb#L26